libraries/androidutils/src/main/kotlin/io/element/android/libraries/androidutils/system/SystemUtils.kt

```diff
@@ -242,6 +242,19 @@ fun startImportTextFromFileIntent(
     }
 }
 
+@Suppress("SwallowedException")
+fun Context.openUrlInExternalApp(
+    url: String,
+    errorMessage: String = getString(R.string.error_no_compatible_app_found),
+) {
+    val intent = Intent(Intent.ACTION_VIEW, Uri.parse(url))
+    try {
+        startActivity(intent)
```

Comment:
I think the Context must be an Activity here, so maybe enforce that?

Reply:
Good point, thanks!